src/main/target/MATEKF722SE/target.h

```diff
@@ -0,0 +1,192 @@
+/*
+ * This file is part of Cleanflight and Betaflight.
+ *
+ * Cleanflight and Betaflight are free software. You can redistribute
+ * this software and/or modify this software under the terms of the
+ * GNU General Public License as published by the Free Software
+ * Foundation, either version 3 of the License, or (at your option)
+ * any later version.
+ *
+ * Cleanflight and Betaflight are distributed in the hope that they
+ * will be useful, but WITHOUT ANY WARRANTY; without even the implied
+ * warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this software.
+ *
+ * If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#define USE_TARGET_CONFIG
+
+#define TARGET_BOARD_IDENTIFIER "MF7S"
+#define USBD_PRODUCT_STRING  "MATEKF722SE"
+
+#define ENABLE_DSHOT_DMAR       true
+
+#define LED0_PIN                PA14  //Blue   SWCLK
+#define LED1_PIN                PA13  //Green  SWDIO
+
+#define USE_BEEPER
+#define BEEPER_PIN              PC13
+#define BEEPER_INVERTED
+
+// *************** SPI1 Gyro & ACC *******************
+
+#define USE_SPI
+#define USE_SPI_DEVICE_1
+#define SPI1_SCK_PIN            PA5
+#define SPI1_MISO_PIN           PA6
+#define SPI1_MOSI_PIN           PA7
+
+#define USE_DUAL_GYRO
+#define USE_EXTI
+#define GYRO_1_EXTI_PIN         PC4
+#define GYRO_2_EXTI_PIN         PC3
+#define MPU_INT_EXTI
+
+#define GYRO_1_CS_PIN           PB2
+#define GYRO_1_SPI_INSTANCE     SPI1
+#define GYRO_2_CS_PIN           PC15
+#define GYRO_2_SPI_INSTANCE     SPI1
+
+#define USE_GYRO
+#define USE_GYRO_SPI_MPU6000
+#define USE_GYRO_SPI_MPU6500
+
+#define USE_ACC
+#define USE_ACC_SPI_MPU6000
+#define USE_ACC_SPI_MPU6500
+
+#define GYRO_MPU6000_1_ALIGN    CW180_DEG_FLIP
```

Comment:
@MATEKSYS These defines are not working anymore with recent master. Gyro and ACC specific alignment parameters are removed some days ago. You should replace this with GYRO_1_ALIGN and so on.

Reply:
No errors when compiling. FC works fine after flashing.  should we must change it?

Reply:
Thanks. I will modify this.

Reply:
@MATEKSYS @fujin for unused defines you will not get any compile errors. The use of this defines was already removed with the related PR and all targets are updated. The actual use will cause your FC to not have the proper alignment with recent master but the firmware will work as usual in all other aspects.

Reply:
https://github.com/betaflight/betaflight/pull/7030